cityofaustin/atd-moped
# add db support for project component phases

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/11427

## Testing
**URL to test:** 

local

**Steps to test:**
- restart hasura cluster and open console
- go to `moped_proj_components` table
- edit an existing record and add a `phase_id` and `subphase_id` (e.g., '1') and populate the `phase_end` and `date_added` fields
- nested data for the associated phase and subphase should be accessible under the `moped_phase` and `moped_subphase` column

---
#### Ship list
